docs/guides/hooks.md

```diff
@@ -0,0 +1,85 @@
+# Hooks
+Hooks exist so that users can latch on to certain Video.js lifecycle events
```

Comment:
Are they similar to events though? You don't generally have the ability to do an action like what we're doing with these hooks.